app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -155,6 +155,7 @@
   private PublishSubject<AppBoughClickEvent> appBought;
   private PublishSubject<String> apkfyDialogConfirmSubject;
   private PublishSubject<Boolean> similarAppsVisibilitySubject;
+  private PublishSubject<DownloadModel.Action> donationsImpressionSubject;
```

Comment:
Since this is also for the downloads, i would recommend changing the name of this subject. I would suggest : "installClickSubject"

Reply:
fixed